website/data/docs-nav-data.json

```diff
@@ -1215,6 +1215,27 @@
   {
     "divider": true
   },
+  {
+    "title": "Release Notes",
+    "routes": [
+       {
+         "title": "Overview",
+         "path": "release-notes"
+       },
+       {
+         "title": "Nomad",
+         "routes": [
+           {
+             "title": "v1.8.x",
+             "path": "release-notes/nomad/v1_8_x"
+           }
+         ]
+       }
+    ] 
+  },
+  {
+    "divider": true
+  },
```

Comment:
For Consul's version of this, this section is at least adjacent to the Upgrades. Maybe we want to move that down here if the goal is to be similar?

Reply:
Does this look ok? If so happy to make the change.

<img width="493" alt="image" src="https://github.com/hashicorp/nomad/assets/551323/76513060-bcda-4d2c-bbee-384747ad5edf">

Reply:
Great!